### DISCUSSION TITLE
Why does PointCloud not have a property ids?

### DISCUSSION THREAD

**Discussion by mxmws** [isl-org/Open3D#6566] [Q&A]
How can I assign IDs to my points? Am I missing something?

**Comment by EwingKang** [comment]
Points in a point cloud are in the form of [core::tensor](https://www.open3d.org/docs/latest/cpp_api/classopen3d_1_1core_1_1_tensor.html), which can be considered as vector/ array. The index to the itself is already a form of "ID". 
What do you need that for? What's your use case?
You can consider using build a [map](https://cplusplus.com/reference/unordered_map/unordered_map/) between some kind of input source and the vector within the PointCloud structure.
